bin/crew

```diff
@@ -447,6 +447,17 @@ def upgrade(*pkgs, build_from_source: false)
 end
 
 def download
+  if ARGV.intersect?(%w[download]) && @pkg.is_fake?
+    return if CREW_FORCE
+    fake_pkg_deplist = @pkg.get_deps_list(return_attr: true).flat_map(&:keys).uniq
+    puts "Will download the following packages: #{fake_pkg_deplist.join(' ')}".orange
+    fake_pkg_deplist.each do |fake_pkg_dep|
+      system "crew download -f #{fake_pkg_dep}"
```

Comment:
Would it be better to avoid recursion and just call the download function outside of system?

Reply:
Agreed. Did just that.